edi_835_parser/loops/transaction.py

```diff
@@ -123,6 +126,16 @@ def other_payee_identification(self) -> Optional[ReferenceSegment]:
 			if ref.qualifier == 'payee identification':
 				return ref
 
+	@property
+	def reference_identification_number(self) -> Optional[ReferenceSegment]:
+		for ref in self.references:
+			if ref.qualifier == 'Reference Identification Number':
+				return ref
```

Comment:
I trust that this is the way. If we have multiple refs, are we wanting to return the moment it hits that value? or should there be a more "hardened" rule on the return. Just checking not ⚔️